plenoptic/simulate/models/frontend.py

```diff
@@ -1,22 +1,18 @@
 import torch
 import torch.nn as nn
 from torchvision import transforms
-
-import numpy as np
 from ...tools.signal import make_disk
-
 import os
-dirname = os.path.dirname(__file__)
+import plenoptic as po
```

Comment:
I think there might be an issue with importing a package inside itself? (specific case I'm thinking of is if they have a version installed via pip and also a local version). I think the relative imports (like `from ...tools.signal import make_disk` above) are safer.

Reply:
Done -- changed to ``from ...tools.display import imshow``